packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -302,79 +314,95 @@ export default class CardSchemaEditor extends Component<Signature> {
                       =
                     </span>
                   {{/if}}
-                  <button
-                    class='pill'
-                    data-test-card-schema-field-navigational-button
-                    {{on 'click' (fn this.openCardDefinition moduleUrl)}}
-                  >
-                    {{#if (this.isLinkedField field)}}
-                      <span class='linked-icon' data-test-linked-icon>
-                        {{svgJar 'icon-link' width='16' height='16'}}
-                      </span>
-                    {{/if}}
-                    <div class='realm-icon'>
-                      <RealmInfoProvider @fileURL={{moduleUrl}}>
-                        <:ready as |realmInfo|>
-                          <img
-                            src={{realmInfo.iconURL}}
-                            alt='Realm icon'
-                            data-test-realm-icon-url={{realmInfo.iconURL}}
-                          />
-                        </:ready>
-                      </RealmInfoProvider>
-                    </div>
-                    <div>
-                      <span>
-                        {{#let
-                          (this.fieldCardDisplayName field.card)
-                          as |cardDisplayName|
-                        }}
-                          <span
-                            data-test-card-display-name={{cardDisplayName}}
-                          >{{cardDisplayName}}</span>
-                        {{/let}}
-                      </span>
-                    </div>
-                  </button>
-                  <DropdownButton
-                    @icon='three-dots-horizontal'
-                    @label='field options'
-                    @contentClass='context-menu'
-                    class='context-menu-trigger'
-                    as |dd|
-                  >
-                    <div class='warning-box'>
-                      <p class='warning'>
-                        These actions will break compatibility with existing
-                        card instances.
-                      </p>
-                      <span class='warning-icon'>
-                        {{svgJar
-                          'warning'
+                  {{#if (this.isOverridden field)}}
+                    {{! template-lint-disable no-invalid-interactive}}
```

Comment:
This linting rule is here for a reason - to prevent accessibility issues (probably a screen reader is not going to know to tell the user that they can click on a `span`)

Reply:
Agree, I used button. [d398580](https://github.com/cardstack/boxel/pull/722/commits/d398580b8f7b9a7843782a9ee4933a43667c6c03)